api/src/main/java/org/pmiops/workbench/config/WebMvcConfig.java

```diff
@@ -77,9 +88,11 @@ public WorkbenchEnvironment workbenchEnvironment() {
    *
    * We may in future rotate key files in production, but will be sure to keep the ones currently
    * in use in cloud environments working when that happens.
+   *
+   * TODO(gjuggler): should we start pulling this file from GCS instead?
    */
   @Lazy
-  @Bean
+  @Bean(name= Constants.GSUITE_ADMIN_CREDS)
```

Comment:
I don't love that we need to include these bean names, but the alternative of creating dumb wrapper classes (GSuiteAdminCreds, which has a single method getGoogleCredential) seemed like more effort than it would be worth.